src/helpers/tokenSectionTypes.ts

```diff
@@ -1,4 +1,6 @@
 export default {
+  bridgeTokenSection: '􀊝 Bridge',
+  crosschainMatchSection: '􀤆 On other networks',
   favoriteTokenSection: '􀋃 Favorites',
   lowLiquidityTokenSection: '􀇿 Low Liquidity',
   unverifiedTokenSection: '􀇿 Unverified',
```

Comment:
not your pr but wondering if we should move it to languages

Reply:
ticketed: https://linear.app/rainbow/issue/TEAM2-525/i18n-token-section-types